modules/administration/pages/ssl-certs-imported.adoc

```diff
@@ -109,11 +113,14 @@ mgrctl exec -ti -- rm /tmp/<Root_CA_Certificate> /tmp/<Server_Cert_File> \
 
 Intermediate CAs can either be available in the file which is specified with `--root-ca-file` or specified as extra options with `--intermediate-ca-file`.
 The `--intermediate-ca-file` option can be specified multiple times.
-This command performs a number of tests on the provided files to test if they are valid and can be used for the requested use case.
+The [command]``mgr-ssl-cert-setup`` command performs a number of tests on the provided files to test if they are valid and can be used for the requested use case.
```

Comment:
I would also mention the main reason to call this tool. It copy/deploy the files to all places where they are needed.